GateWatcher/aioniq_ecs/ingest/parser.yml

```
@@ -0,0 +1,768 @@
+  flow:
+    actions:
+      - set:
+          gatewatcher.flow.start: '{{json_load.message.flow.start}}'
+          gatewatcher.flow.pkts_toclient: '{{json_load.message.flow.pkts_toclient}}'
+          gatewatcher.flow.pkts_toserver: '{{json_load.message.flow.pkts_toserver}}'
+          gatewatcher.flow.bytes_toclient: '{{json_load.message.flow.bytes_toclient}}'
+          gatewatcher.flow.bytes_toserver: '{{json_load.message.flow.bytes_toserver}}'
+        filter: ''
+        name: set
+  shellcode:
+    actions:
+      - set:
+          gatewatcher.shellcode.id: '{{json_load.message.shellcode.id}}'
+          gatewatcher.shellcode.analysis: '{{json_load.message.shellcode.analysis}}'
+          gatewatcher.shellcode.sub_type: '{{json_load.message.shellcode.sub_type}}'
+          gatewatcher.shellcode.encodings: '{{json_load.message.shellcode.encodings}}'
+          gatewatcher.shellcode.sample_id: '{{json_load.message.shellcode.sample_id}}'
+        filter: ''
+        name: set
+  malicious_powershell:
+    actions:
+      - set:
+          gatewatcher.malicious_powershell.id: '{{json_load.message.malicious_powershell.id}}'
+          gatewatcher.malicious_powershell.score: '{{json_load.message.malicious_powershell.score}}'
+          gatewatcher.malicious_powershell.sample_id: '{{json_load.message.malicious_powershell.sample_id}}'
+          gatewatcher.malicious_powershell.score_details: '{{json_load.message.malicious_powershell.score_details}}'
+          gatewatcher.malicious_powershell.proba_obfuscated: '{{json_load.message.malicious_powershell.proba_obfuscated}}'
+        filter: ''
+        name: set
+  files:
+    actions:
+      - set:
+          gatewatcher.files: '{{json_load.message.files}}'
+        filter: ''
+        name: set
+  history:
+    actions:
+      - set:
+          gatewatcher.history.id: '{{json_load.message.history.id}}'
+          gatewatcher.history.ip: '{{json_load.message.history.ip}}'
+          gatewatcher.history.code: '{{json_load.message.history.code}}'
+          gatewatcher.history.name: '{{json_load.message.history.name}}'
+          gatewatcher.history.type: '{{json_load.message.history.type}}'
+          gatewatcher.history.method: '{{json_load.message.history.method}}'
+          gatewatcher.history.content: '{{json_load.message.history.content}}'
+          gatewatcher.history.endpoint: '{{json_load.message.history.endpoint}}'
+        filter: ''
+        name: set
+  syslog:
+    actions:
+      - set:
+          log.syslog.appname: '{{json_load.message.log.syslog.appname}}'
+          log.syslog.priority: '{{json_load.message.log.syslog.priority}}'
+          log.syslog.facility.code: '{{json_load.message.log.syslog.facility.code}}'
+          log.syslog.facility.name: '{{json_load.message.log.syslog.facility.name}}'
+          log.syslog.severity.code: '{{json_load.message.log.syslog.severity.code}}'
+          log.syslog.severity.name: '{{json_load.message.log.syslog.severity.name}}'
```

These fields are set by our workflow with the information get from the syslog message used to forward the event.
I suggest prefixing these fields or using ECS alternative (when possible):
```suggestion
          network.application: '{{json_load.message.log.syslog.appname}}'
          gatewatcher.syslog.priority: '{{json_load.message.log.syslog.priority}}'
          gatewatcher.syslog.facility.code: '{{json_load.message.log.syslog.facility.code}}'
          gatewatcher.syslog.facility.name: '{{json_load.message.log.syslog.facility.name}}'
          event.severity: '{{json_load.message.log.syslog.severity.code}}'
          gatewatcher.syslog.severity.name: '{{json_load.message.log.syslog.severity.name}}'
```